shared-module/rgbmatrix/RGBMatrix.c

```diff
@@ -180,9 +180,6 @@ void common_hal_rgbmatrix_rgbmatrix_deinit(rgbmatrix_rgbmatrix_obj_t* self) {
 
 void rgbmatrix_rgbmatrix_collect_ptrs(rgbmatrix_rgbmatrix_obj_t* self) {
     gc_collect_ptr(self->framebuffer);
-    gc_collect_ptr(self->protomatter.rgbPins);
-    gc_collect_ptr(self->protomatter.addr);
-    gc_collect_ptr(self->protomatter.screenData);
```

Comment:
Question: Are these no longer needed because supervisor allocated regions get normal GC?

Reply:
Correct (unless there are bugs). As long as they live on the GC heap, they are covered by the linked list starting from `MP_STATE_VM(first_embedded_allocation)`. Once they are moved off it, they don’t need it anymore.